### PR TITLE
Add "create user" tutorial

### LINKED ISSUE #27: Add 'Create User' to Tutorials / Rework First Use
In order to also follow the tutorials, when running CrateDB on a remote host, I think it would be good to a simple tutorial for adding one user.

**Changes**:
- add **Create User** guide to tutorials
- add **Create User** Tutorial to index
- Rework **First Use** to reference **Create User**
- Changed "admin UI" to "Admin UI" in **First Use**
- Minor formating changes within **First Use**

### PR DESCRIPTION
**Pull request by proddata**
## Summary of the changes / Why this is an improvement

- add Create User guide to tutorials
- add Create User Tutorial to index
- Rework First Use to reference Create User
- Changed "admin UI" to "Admin UI" in First Use
- Minor formating changes within First Use

closes #27 

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Review by infoverload — APPROVED** [review]

**Comment by amotl** [comment]
Hi Georg,

I just rebased the patch for this excellent improvement on top of the current master branch and added some further edits through e424aff, along the lines of removing any references to the former "enterprise features". I hope you like them.

With kind regards,
Andreas.

**Comment by nomicode** [comment]
@amotl @proddata do you mind if I also give this an editing pass and push a commit?

**Comment by proddata** [comment]
please do :)

**Comment by nomicode** [comment]
as noted elsewhere: I am on vacation next week. I will come back to this afterward and start work. have a nice Easter weekend both of you

**Comment by amotl** [comment]
Hi Naomi,

we believe this is good to go and we dearly want it to hit our documentation space.

> I am on vacation next week. I will come back to this afterward and start work.

I will be happy to see some copyedits from your pen on behalf of another iteration. Enjoy your vacation!

With kind regards,
Andreas.

File: docs/create-user.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+------------
+Introduction
+------------
+
+This part of the documentation sheds some light on the topics of `user
+management`_ and `privileges`_.
+
+CrateDB ships with a superuser account called "``crate``", which has the
+privileges to perform any action. However, with the default configuration, this
+superuser can only access CreateDB from the local machine CrateDB has been
```
Comment (seut) [inline docs/create-user.rst:17]: There is a small typo `CreateDB` ;)

Comment (proddata) [reply docs/create-user.rst:17]: https://github.com/crate/crate-tutorials/pull/58 :)

**Pull request merged**